### PR TITLE
Fix Dataset Dates

### PR DESCRIPTION
**Pull request by bhyeh**
1. Update 2020 corrective Ethiopia Tigray dataset start and end dates (24 months).
2. Update 2021 Ghent University Ethiopia dataset end date (2022-12-31)
3. Deleted above datasets from `data/datasets` and added updated `datasets.dvc` for regeneration of the two

### CODE REVIEW

**Comment by ivanzvonkov** [comment]
@bhyeh this looks good, please let me know when the datasets have been regenerated and rerun the tests using https://github.com/nasaharvest/crop-mask/wiki/3.-Adding-labeled-data#6-checking-progress-of-the-dataset

**Comment by bhyeh** [comment]
> @bhyeh this looks good, please let me know when the datasets have been regenerated and rerun the tests using https://github.com/nasaharvest/crop-mask/wiki/3.-Adding-labeled-data#6-checking-progress-of-the-dataset

:+1: looks like pipeline is still pipelining. Ghent Ethiopia is done but Ethiopia corrective is still exporting it looks. I'll retrigger again EOD + rerun tests if its completed then.

**Comment by bhyeh** [comment]
> > @bhyeh this looks good, please let me know when the datasets have been regenerated and rerun the tests using https://github.com/nasaharvest/crop-mask/wiki/3.-Adding-labeled-data#6-checking-progress-of-the-dataset
> 
> 👍 looks like pipeline is still pipelining. Ghent Ethiopia is done but Ethiopia corrective is still exporting it looks. I'll retrigger again EOD + rerun tests if its completed then.

Looks like I actually miscalculated the start and end dates for the corrective Ethiopia and added another 12 months. Do you think this will be an issue that would require fixing now?

**Comment by ivanzvonkov** [comment]
Every time the code for processing a dataset in `datasets.py` is updated, the respective dataset csv in `data/datasets` has to be deleted so it's properly regenerated.

File: data/report.txt [hunk 1]
```diff
@@ -364,7 +364,7 @@ eo_data_complete    47
 
 
 
-EthiopiaTigrayCorrective2020 (Timesteps: 12)
+EthiopiaTigrayCorrective2020 (Timesteps: 24)
 ----------------------------------------------------------------------------
 eo_data_complete    200
 ✔ training amount: 200, positive class: 2.0%
```
Comment (bhyeh) [inline data/report.txt:370]: completed! I went back to delete and fix the time steps here and ran the pipeline again. ready for your eyes to look over @ivanzvonkov

**Review by ivanzvonkov — APPROVED** [review]
Thanks @bhyeh, all checks now pass, I'll go ahead and merge.

**Pull request merged**